modules/gathering-data-specific-features.adoc

```diff
@@ -40,6 +40,9 @@ ifndef::openshift-origin[]
 ifndef::openshift-rosa,openshift-dedicated[]
 |`registry.redhat.io/odf4/odf-must-gather-rhel9:v<installed_version_ODF>`
 |Data collection for {rh-storage-first}.
+
+|`quay.io/redhat-user-workloads/cluster-observabilit-tenant/cluster-observability-operator-container`
```

Comment:
Image name should be
registry.redhat.io/cluster-observability-operator/cluster-observability-rhel8-operator

Reply:
I have updated the image